mix.exs

```diff
@@ -34,8 +34,10 @@ defmodule ApplicationRunner.MixProject do
     [
       {:credo, "~> 1.6.7", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.2", only: [:dev, :test], runtime: false},
-      {:ex_component_schema,
-       git: "https://github.com/lenra-io/ex_component_schema", ref: "v1.0.0-beta.3"},
+      {
+        :ex_component_schema,
+        git: "https://github.com/lenra-io/ex_component_schema", ref: "21adb5c"
```

Comment:
Not up to date

Reply:
We need to wait this PR https://github.com/lenra-io/ex_component_schema/pull/12 merge and switch to beta branch